README.md

```diff
@@ -0,0 +1,11 @@
+# Page Builder modules CLI
+
+This is a CLI for scaffolding both custom and extension modules for Page Builder content types.
+
+## Usage
+1. Clone or download the repo.
+1. CD into the pbmodules directory and run `npm install -g`.
```

Comment:
should be `npm install`. `g` flag for installing dependencies globally

Reply:
RIght, but I think running `npm install -g` runs npm install for the packages and the `-g` flag installs `pbmodules` globaly so it can be executed within the destination folder for the module.